pkg/monitoring/metrics/forklift-controller/migration_metrics.go

```diff
@@ -0,0 +1,110 @@
+package forklift_controller
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Calculate Migrations metrics every 10 seconds
+func RecordMigrationMetrics(c client.Client) {
+	go func() {
+		for {
+			time.Sleep(10 * time.Second)
```

Comment:
does 10 sec make sense? or should it be a bigger number? (how many plan/migration can change in 10 sec?) unless it suppose to be pretty real time accurate?

Reply:
I followed the time the previous team used, I guess its less for changes in exiting migration/plans but more for new ones, but 30 sec should also be fine?